{"timestamp":"2025-09-01T03:41:10.372075","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-09-01T03:41:10.372799","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/lesson6_sensor.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-09-01T03:41:11.117536Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-01T03:41:11.117771Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-01T03:41:11.118003Z","level":"info","event":"Current task name:wait_for_file","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-01T03:41:11.118269Z","level":"info","event":"Dag name:lesson6_sensor","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-01T03:41:11.121410","level":"warning","event":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py:471: DeprecationWarning: Using Connection.get_connection_from_secrets from `airflow.models` is deprecated.Please use `from airflow.sdk import Connection` instead\n  warnings.warn(\n","logger":"py.warnings"}
{"timestamp":"2025-09-01T03:41:11.131982","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AirflowNotFoundException","exc_value":"The conn_id `fs_default` isn't defined","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":890,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1177,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/sensors/filesystem.py","lineno":143,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/sensor.py","lineno":227,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/sensor.py","lineno":207,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/sensors/filesystem.py","lineno":129,"name":"poke"},{"filename":"/usr/local/lib/python3.12/functools.py","lineno":998,"name":"__get__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/sensors/filesystem.py","lineno":123,"name":"path"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/hooks/filesystem.py","lineno":64,"name":"__init__"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py","lineno":64,"name":"get_connection"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py","lineno":478,"name":"get_connection_from_secrets"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/definitions/connection.py","lineno":147,"name":"get"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-09-01T03:41:11.133933Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-01T03:41:11.134194Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-01T03:41:11.134454Z","level":"info","event":"Task:<Task(FileSensor): wait_for_file>","chan":"stdout","logger":"task"}
{"timestamp":"2025-09-01T03:41:11.134727Z","level":"info","event":"Failure caused by The conn_id `fs_default` isn't defined","chan":"stdout","logger":"task"}
